chapters/ch48.md

```diff
@@ -1,55 +1,57 @@
 
---> _replace THIS LINE by your translation for the above line_
+Ngược lại, một hệ thống hoc bắt đầu - kết thúc đọc đầu vào một audio clip, và cố gắng để cho kêt quả trực tiếp đầu ra là bản dịch:
 
 ![img](../imgs/C48_02.png)
 
 > So far, we have only described machine learning "pipelines" that are completely linear: the output is sequentially passed from one staged to the next. Pipelines can be more complex. For example, here is a simple architecture for an autonomous car:
 
---> _replace THIS LINE by your translation for the above line_
+Từ đầu đến giờ, chúng ta mới mô tả các "chuỗi thuật toán" sử dụng trong học máy hoàn toàn tuyến tính: đầu ra được chuyền đổi theo dạng chuổi từ một trạng thái sang trạng thái tiếp theo. Các "chuỗi thuật toán" này có thể phức tạp hơn. Ví dụ dưới đây là mọt pipline cho một kiến trúc đơn giản của hệ thống lái xe tự động:
 
 ![img](../imgs/C48_03.png)
 
 > It has three components: One detects other cars using the camera images; one detects pedestrians; then a final component plans a path for our own car that avoids the cars and pedestrians.
 
---> _replace THIS LINE by your translation for the above line_
+Kiến trúc này gồm ba phần: phần một phát hiện các xe khác trên đường sử dụng các hình ảnh từ máy quay; phần hai phát hiện người đi bộ; và phần cuối cùng vạch ra đường đi cho chiếc xe để tránh các xe khác cũng như những người đi bộ trên đường. 
 
 > Not every component in a pipeline has to be learned. For example, the literature on "robot motion planning" has numerous algorithms for the final path planning step for the car. Many of these algorithms do not involve learning.
 
---> _replace THIS LINE by your translation for the above line_
+Không phải mọi phần trong một chuỗi thuật toán phải được học. Ví dụ, các nghiên cứu về "quỹ đạo chuyển động robot" sử dụng rất nhiều thuật toán cho bước cuối cùng, bước xác định quỹ đạo đường đi của chiếc xe. Rất nhiều thuật toán này không liên quan đến việc học.   
 
 > In contrast, and end-to-end approach might try to take in the sensor inputs and directly output the steering direction:
 
---> _replace THIS LINE by your translation for the above line_
+Ngược lại, phương pháp hoc bắt đầu - kết thúc có khả năng để lấy đầu vào từ hệ thống cảm biến, và cho trực tiếp kết quả hướng lái ở đầu ra. 
 
 ![img](../imgs/C48_04.png)
 
 > Even though end-to-end learning has seen many successes, it is not always the best approach. For example, end-to-end speech recognition works well. But I’m skeptical about end-to-end learning for autonomous driving. The next few chapters explain why.
 
---> _replace THIS LINE by your translation for the above line_
+Mặc dầu phương pháp hoc bắt đầu - kết thúc đã cho ra nhiều kết quả thành công, nó không phải luôn luôn là phương pháp tốt nhât. Ví dụ, trong bài toán  nhận diện giọng nói phương pháp hoc bắt đầu - kết thúc hoạt động tốt, nhưng tôi luôn nghi ngờ việc hoc bắt đầu - kết thúc trong xe lái tự động. Các chương tiếp theo sẽ giải thích tại sao. 
```

Comment:
"Mặc dầu" -> "mặc dù".

Reply:
Mình sẽ cố gắng sửa lại phần dịch của mình, và reply trước ngày 24/11.

Reply:
"bài toán  nhận diện giọng nói" dư một khoảng trắng nhé.

Reply:
> Mình sẽ cố gắng sửa lại phần dịch của mình, và reply trước ngày 24/11.

@ngoclinh84phys bạn có đang tiếp tục làm việc với PR này không? Nếu hết ngày hôm nay bạn không trả lời thì mình sẽ đóng PR này.